src/DependencyInjection/Configuration.php

```diff
@@ -303,4 +304,32 @@ protected function getDashboardNode()
 
         return $node;
     }
+
+    protected function getDefaultsNode()
+    {
+        $builder = new TreeBuilder();
+        $node = $builder->root('defaults');
+
+        $node
+            ->requiresAtLeastOneElement()
+                ->useAttributeAsKey('key')
+                ->arrayPrototype()
```

Comment:
This whole section looks like Copy&Paste and partially wrong. Please use `getLanguagesNode()` or `getThemeNode()` as template.

Reply:
Sorry, I'd not used this config specification mechanism before.  I home the new code is right.

Reply:
Well, actually I think it should be:
```php
        $builder = new TreeBuilder();
        $node = $builder->root('defaults');

        $node
            ->addDefaultsIfNotSet()
            ->children()
                 ->scalarNode('timezone')->defaultValue('Europe/Berlin')->end()
                 ->scalarNode('country')->defaultValue('DE')->end()
                 ->scalarNode('currency')->defaultValue('EUR')->end()
            ->end()
        ;

        return $node;
```